src/core/ui.js

```diff
@@ -114,6 +114,58 @@ function createWarnButton(butName, arr, title) {
     ["label", `Document ${title.toLowerCase()}`],
   ]);
   ariaDecorate(button, ariaMap);
+
+
+  var numSteps = 20.0;
+  var boxElement
+  var prevRatio = 0.0;
+  var increasingColor = "rgba(255, 255, 255)";
+  var decreasingColor = "rgba(255, 255, 255, ratio)";
```

Comment:
I like this idea of changing the alpha. CSS variables might be worth looking into here :D

Reply:
Ohk! I'll have a look at it...

Reply:
Probably can trigger a simple CSS transition to fade to 50% when intesecting. Hovering or otherwise activating the button should make it fade back in, even if over the logo.

Reply:
@marcoscaceres Yeah that could be done I'll implement it